plugins/protocolbuffers/pyi/v25.0/buf.plugin.yaml

```diff
@@ -14,5 +14,5 @@ registry:
     requires_python: ">=3.8"
     deps:
       # https://pypi.org/project/protobuf/
-      - "protobuf"
+      - "protobuf~=4.25"
       - "types-protobuf"
```

Comment:
FWIW, types-protobuf seemingly follows a similar versioning scheme, but it seems like the PRs hang out a lot longer in typeshed: https://github.com/python/typeshed/pull/12026. So if we wanted to specify a range, we'd probably have to wait until those upstream PRs land.